site/docs/tools/spark_auth_config.md

```diff
@@ -0,0 +1,55 @@
+# Authentication in Spark
```

Comment:
I guess would be also needed for Flink and Hive, isn't? If yes, I think in my opinion one page called `Authentication` that includes subsection about the authentication for every integration would make sense.

Reply:
Definitely good to have those as well.

Reply:
I do not really know how to configure Hive or Flink.. @omarsmak : Do we have any other pages about those environments?

Reply:
@dimas-b for hive I am planning to add the documentation next week perhaps which will make it clear how to configure. Maybe next week I can as well add the hive auth settings here as well. 
For Flink, I think you can take a look at the Flink demo to see how is it done. 
I think for now you can just have the page structure ready for more integrations sections to add.